### DISCUSSION TITLE
Railway部署后，客户端怎么报Response 401 或200错误

### DISCUSSION THREAD

**Discussion by chenmo-tssz** [zdz/ServerStatus-Rust#199] [Q&A] [answered]
https://serverstatus-rust-production-b1ff.up.railway.app/report 
用的默认配置
我用两台客户端测试一下，一个报Response 401  一个报Response 200  是我哪里没配置对吗？

**Comment by zdz** [comment]
401 是 http 的认证错误，检查一下 name/pass 相关的

**Reply by chenmo-tssz** [reply]
都是用的默认配置     今天看日志提示 ERROR stat_server::stats > report error => Error("missing field `timezone`", line: 0, column: 0)  这是时区没设置对吗？

**Reply by zdz** [reply]
升级到最新版本

**Reply by chenmo-tssz** [reply]
是指客户端吗？ 都是按着README.md的操作的  我去检查一下先。

**Reply by zdz** [reply]
都升级

**Reply by chenmo-tssz** [reply]
怎么看版本是多少？

**Reply by chenmo-tssz** [reply, marked as answer]
手动下最新版解决了。 https://raw.githubusercontent.com/zdz/ServerStatus-Rust/master/client/stat_client.py 这个不是最新的客户端。
